shippable.yml

```diff
@@ -0,0 +1,45 @@
+language: c
+
+compiler:
+  - gcc
+  - clang
+
+env:
+  matrix:
+    - CONF=""
+    # - CONF="--disable-abi-compat"
+    # - CONF="--enable-schedule-sp"
+    # - CONF="--enable-schedule-iquery"
+    # - CONF="--enable-dpdk-zero-copy"
+    - CROSS_ARCH="arm64"
```

Comment:
It seems there is a confusion here. Currently this setup generates 4 build jobs (2 for CONF setting, 2 for CROSS_ARCH). There is no need to run CROSS_ARCH jobs (and it is not a cross-build anyway). Let's drop CROSS_ARCH settings altogether.

Reply:
My understanding is that this shippable.yml will evolve shortly to perform the same build functionality in the .travis.yml config, including the cross-arch testing. This is simply the initial build sample to start with a simple example, that will then proceed to add in the additional tests. As such, I suggest leaving this in as a placeholder for these next steps.

Reply:
yep, I agree to have it as it now. We need to make some simple run. Then extend it. It will be nice if we can have common file for Shippable and Travis or generate them from some common template. But that can be later.

Reply:
@ttrahan @muvarov ATM this uncomented CROSS_ARCH lines does not make sense. I'd prefer to see it commented, so that there are no extra builds in Shippable.

Reply:
@lumag: agreed. I've commented out the unused CROSS_ARCH variable and the unused global variable.

Reply:
@ttrahan Thanks! It now looks fine for me.